modules/internal/fixInternalDocs.sh

```diff
@@ -62,7 +62,7 @@ function fixTitle() {
   # Module: ChapelFoo
   local titleLen=${#1}
   local header="$(printf '=%.0s' $(seq 1 $titleLen))"
```

Comment:
I think you can remove these two local vars and the comment above them, they will no longer be used.